src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -2410,6 +2410,18 @@ public override AwaitExpressionInfo GetAwaitExpressionInfo(AwaitExpressionSyntax
             return memberModel == null ? default(AwaitExpressionInfo) : memberModel.GetAwaitExpressionInfo(node);
         }
 
+        public override AwaitExpressionInfo GetAwaitExpressionInfo(LocalDeclarationStatementSyntax node)
+        {
+            MemberSemanticModel memberModel = GetMemberModel(node);
+            return memberModel == null ? default(AwaitExpressionInfo) : memberModel.GetAwaitExpressionInfo(node);
+        }
+
+        public override AwaitExpressionInfo GetAwaitExpressionInfo(UsingStatementSyntax node)
+        {
+            MemberSemanticModel memberModel = GetMemberModel(node);
+            return memberModel == null ? default(AwaitExpressionInfo) : memberModel.GetAwaitExpressionInfo(node);
```

Comment:
>memberModel.GetAwaitExpressionInfo(node) [](http://example.com/codeflow?start=72&length=40)

Is this code path covered by a test?

Reply:
Yes, the existing tests go through here.